insightconnect_plugin_runtime/api/endpoints.py

```diff
@@ -568,29 +568,37 @@ def connection_test():
                   description: Internal server error
             """
             status_code = 200
-            message = None
+            output = None
+
+            input_message = request.get_json(force=True)
+            self.logger.debug("Request input: %s", input_message)
```

Comment:
```suggestion
            self.logger.debug(f"Request input: {input_message}")
```
You may want to get rid of this, though - in production this may log credentials/secrets.